hackathon_site/hackathon_site/settings/__init__.py

```diff
@@ -136,6 +136,8 @@
     }
 }
 
+DEFAULT_AUTO_FIELD = "django.db.models.AutoField"
```

Comment:
why do we need to change this?

Reply:
I think this is used to avoid unwanted migrations in the future (Saw it in Django doc) @YuYing-Liang

Reply:
https://docs.djangoproject.com/en/3.2/releases/3.2/#customizing-type-of-auto-created-primary-keys